UI/PongWars/src/UnoPongWars/Presentation/GamePage.cs

```diff
@@ -18,52 +18,52 @@ public GamePage()
-                            .ItemTemplate<Cell>(cell =>
-                                new Grid()
-                                .Children(
-                                    new Rectangle()
-                                        .Width(10)
-                                        .Height(10)
-                                        .Fill(x => x.Binding(() => cell)
-                                            .Convert(cell => new SolidColorBrush(CellColor(cell)))),
-                                    new Ellipse()
-                                        .Width(10)
-                                        .Height(10)
-                                        .Fill(x => x.Binding(() => cell)
-                                            .Convert(cell => new SolidColorBrush(PlayerColor(cell))))
-                                        .Visibility(x => x.Binding(() => cell)
-                                            .Convert(cell => cell.HasBall ? Visibility.Visible : Visibility.Collapsed))))),
-                        new TextBlock()
-                            .Text(x => x.Binding(() => vm.Score))
+                    .Margin(20)
+                    .RowDefinitions("*,auto,auto")
+                    .Children(
+                        new Viewbox()
                             .HorizontalAlignment(HorizontalAlignment.Center)
-                            .Style(Theme.TextBlock.Styles.HeadlineSmall)
-                            .Margin(10)
-                            .Grid(row: 1),
-                        new Slider()
-                            .MaxWidth(400)
-                            .Margin(16)
-                            .Maximum(1000)
+                            .Child(
+                                new ItemsRepeater()
+                                    .ItemsSource(() => vm.Cells)
+                                    .VerticalAlignment(VerticalAlignment.Center)
+                                    .Width(160)
+                                    .Height(160)
+                                    .Layout(
+                                        new UniformGridLayout()
+                                            .Orientation(Orientation.Vertical)
+                                            .MaximumRowsOrColumns(16))
+                                    .ItemTemplate<Cell>(cell => CellTemplate(cell))),
+                                new TextBlock()
+                                    .Text(() => vm.Score)
+                                    .HorizontalAlignment(HorizontalAlignment.Center)
+                                    .Style(Theme.TextBlock.Styles.HeadlineSmall)
+                                    .Margin(10)
+                                    .Grid(row: 1),
+                                new Slider()
+                                    .MaxWidth(400)
+                                    .Margin(16)
+                                    .Maximum(1000)
 #if __ANDROID__
```

Comment:
@dr1rrb, @morning4coffe-dev I think we would need to add a comment here to give a bit more info on why we have a `#if __ANDROID__` here